ledger/storage/index/memory_cmp_test.go

```diff
@@ -0,0 +1,102 @@
+/*
+ *    Copyright 2019 Insolar Technologies
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package index_test
+
+import (
+	"math/rand"
+	"testing"
+
+	fuzz "github.com/google/gofuzz"
+	"github.com/insolar/insolar/core"
+	"github.com/insolar/insolar/gen"
+	"github.com/insolar/insolar/instrumentation/inslogger"
+	"github.com/insolar/insolar/ledger/storage/index"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInMemoryIndex(t *testing.T) {
+	t.Parallel()
+
+	ctx := inslogger.TestContext(t)
+
+	indexStorage := index.NewStorageMemory()
+
+	type tempIndex struct {
```

Comment:
If I understand the code correctly, "ls" and "pn" here are not required. We can write directly into idx variable.

Reply:
Right.